File: C++/Types of Matrix/Matrix subtraction.cpp
#include <iostream>
#include <vector>
using namespace std;

int main(){
    vector <vector<int>> A = {
        { 10, 20, 30 },
        { 40, 50, 60 },
        { 70, 80, 90 }
    };
    vector <vector<int>> B = {
        { 1, 2, 3 },
        { 4, 5, 6 },
        { 7, 8, 9 }
    };

    if(A.size() == B.size() && A[0].size() == B[0].size()){
        cout << "Matrix (after subtraction): " << endl;
        for (int row = 0; row < A.size(); row++) {
            for (int col = 0; col < A[row].size(); col++) {
                cout << "\t" << (A[row][col] - B[row][col]);
            }
            cout << endl;
        }
    }else{
        cout << "Error! Number of rows and columns must be same." << endl;
    }
}

File: Python/Types of Matrix/Matrix subtraction.py
A = [[10, 20, 30],
     [40, 50, 60],
     [70, 80, 90]]
B = [[1, 2, 3],
     [4, 5, 6],
     [7, 8, 9]]

if len(A) == len(B) and len(A[0]) == len(B[0]):
    print("Matrix (after subtraction): ")
    for row in range(len(A)):
        for col in range(len(A[row])):
            print("\t", (A[row][col] - B[row][col]), end='')
        print()
else:
    print("Error! Number of rows and columns must be same.")